charts/rancher-cloud/values.yaml

```diff
@@ -0,0 +1,42 @@
+# Default values for rancher-cloud.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+global:
+  aws:
+    enabled: false
+    accountNumber: ""
+    roleName: ""
+
```

Comment:
I think the global.csp entries are missing in the values.yaml

Reply:
We have a separate csp_values.yaml file that has those entries. Maybe we can combine them at some point if we have a boolean to enable the values for a specific-csp.